packages/react-devtools-shell/src/app/SuspenseTree/index.js

```diff
@@ -25,10 +25,15 @@ function SuspenseTree() {
       <PrimaryFallbackTest initialSuspend={true} />
       <NestedSuspenseTest />
       <SuspenseListTest />
+      <EmptySuspense />
     </Fragment>
   );
 }
 
+function EmptySuspense() {
+  return <Suspense />;
+}
```

Comment:
Technically this fixture doesn't help here since React and DevTools are in-sync, but it seemed nice to cover this missing case for the future.